Code/Framework/AzFramework/AzFramework/Physics/Configuration/JointConfiguration.h

```diff
@@ -66,4 +72,18 @@ namespace AzPhysics
         // Default all visibility settings to invisible, since most joint configurations don't need to display these.
         AZ::u8 m_propertyVisibilityFlags = 0;
     };
-}
+
+    //! Default colors, line widths etc for use when visualizing joints.
+    namespace JointVisualizationDefaults
+    {
+        const float Alpha = 0.6f;
+        const AZ::Color ColorDefault = AZ::Color(1.0f, 1.0f, 1.0f, Alpha);
+        const AZ::Color ColorFirst = AZ::Color(1.0f, 0.0f, 0.0f, Alpha);
+        const AZ::Color ColorSecond = AZ::Color(0.0f, 1.0f, 0.0f, Alpha);
+        const AZ::Color ColorSweepArc = AZ::Color(1.0f, 1.0f, 1.0f, Alpha);
```

Comment:
suggestion: It might also be worth making these AZCVARs so that designers/UX can more easily tweak them without needing input from an engineer. You can extern them here (there's a macro for this) and then define them in the .cpp file and then designers can tweak them from the console or a cfg file)

Reply:
Yeah, I think that's a nice idea but I'll leave it for after the UX review.